### PR TITLE
[Builder] Add documentation

### PR DESCRIPTION
**Pull request by makrushinoff**
Solves #104 issue

### CODE REVIEW

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 1]
```diff
@@ -14,78 +14,217 @@
...
+     * @return this object
+     */
     public EventBuilder setPlace(Place place) {
         this.place = place;
         return this;
     }
 
+    /**
+     * Method sets invited {@link EventBuilder#invited} and returns
+     * current EventBuilder object
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:118]: Add '.' at the end

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 2]
```diff
@@ -14,78 +14,217 @@
...
+     * @return this object
+     */
     public EventBuilder setInvited(List<Human> invited) {
         this.invited = invited;
         return this;
     }
 
+    /**
+     * Method sets date {@link EventBuilder#date} and returns
+     * current EventBuilder object
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:130]: Add '.' at the end

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 3]
```diff
@@ -14,78 +14,217 @@
...
+     * @return this object
+     */
     public EventBuilder setDate(LocalDate date) {
         this.date = date;
         return this;
     }
 
+    /**
+     * Method sets time {@link EventBuilder#time} and returns
+     * current EventBuilder object
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:142]: Add '.' at the end

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 4]
```diff
@@ -14,78 +14,217 @@
...
+     * @return this object
+     */
     public EventBuilder setTime(LocalTime time) {
         this.time = time;
         return this;
     }
 
+    /**
+     * Method sets priority {@link EventBuilder#priority} and returns
+     * current EventBuilder object
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:154]: Add '.' at the end

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 5]
```diff
@@ -14,78 +14,217 @@
...
+     * @return this object
+     */
     public EventBuilder setPriority(Priority priority) {
         this.priority = priority;
         return this;
     }
 
+    /**
+     * Method sets tags {@link EventBuilder#tags} and returns
+     * current EventBuilder object
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:166]: Add '.' at the end

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 6]
```diff
@@ -14,78 +14,217 @@
...
+     * @return this object
+     */
     public EventBuilder setTags(List<Tag> tags) {
         this.tags = tags;
         return this;
     }
 
+    /**
+     * Method sets duration {@link EventBuilder#duration} and returns
+     * current EventBuilder object
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:178]: Add '.' at the end

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 7]
```diff
@@ -14,78 +14,217 @@
...
+     * @return this object
+     */
     public EventBuilder setDuration(LocalTime duration) {
         this.duration = duration;
         return this;
     }
 
+    /**
+     * Method sets links {@link EventBuilder#links} and returns
+     * current EventBuilder object
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:190]: Add '.' at the end

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 8]
```diff
@@ -14,78 +14,217 @@
...
+     *
+     * @return new Event object
+     * @see Event
+     */
     public Event build() {
         return new Event(id, name, place, invited, date, time, priority, tags, duration, links, types);
     }
 
+    /**
+     * Method sets own fields from input Event
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:223]: Add '.' at the end

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 9]
```diff
@@ -14,78 +14,217 @@
...
+     * @param types - types field
+     * @return this object
+     */
     public EventBuilder setTypes(Set<Type> types) {
         this.types = types;
         return this;
     }
 
+    /**
+     * Method create Event object using it`s own fields
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:213]: Add '.' at the end

File: src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java [hunk 10]
```diff
@@ -14,78 +14,217 @@
...
+     * @return this object
+     */
     public EventBuilder setLinks(List<Link> links) {
         this.links = links;
         return this;
     }
 
+    /**
+     * Method sets types {@link EventBuilder#types} and returns
+     * current EventBuilder object
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/builder/EventBuilder.java:202]: Add '.' at the end

**Review by MLGMAG — CHANGES_REQUESTED** [review]

**Review by MLGMAG — APPROVED** [review]

**Review by VasiliyPetrov1 — APPROVED** [review]

**Pull request merged**